mzLib/MzLibUtil/MrsNoiseEstimation/MRSNoiseEstimator.cs

```diff
@@ -25,7 +23,7 @@ public static class MRSNoiseEstimator
         {
             int iterations = 0; 
             // 1. Estimate the standard deviation of the noise in the original signal. 
-            double stdevPrevious = BasicStatistics.CalculateStandardDeviation(signal);
+            double stdevPrevious = signal.StandardDeviation();
```

Comment:
how about naming this EstimatedStandardDeviation if it is not actually the standard deviation